lib/components/app/responsive-webapp.js

```diff
@@ -194,9 +195,16 @@ class ResponsiveWebapp extends Component {
   renderDesktopView = () => {
     const { sessionTimeoutSeconds } = this.props
     const { MainControls, MainPanel, MapWindows } = this.context
+    const { popupContent, setPopupContent } = this.props
     return (
       <div className="otp">
         <DesktopNav />
+        {popupContent && (
```

Comment:
Could the check for non-null `popupContent` be done inside `PopupWrapper`?

Reply:
This is much cleaner thanks!